assets/css/edd-admin.css

```diff
@@ -230,6 +232,11 @@ a.edd-delete:hover {
 	z-index: 9999 !important;
 }
 
+@media screen and (max-width: 782px) {
```

Comment:
I wonder if the custom margin added to `#edd-filters .filter-items .edd-date-range-options` (line 2461) could be removed so the spans here could all be styled together, so you are not having to override them? Also see that there are some `inline-block` styles applied to these elements which may not be necessary since the parent container is using flexbox.

Reply:
I think I made the specificity in this media query too specific, since the `.edd-date-range-options` isn't actually used here.  

It looks like it is specific to reports and as far as I can tell, `#edd-filters .filter-items .edd-date-range-options` isn't used anywhere.  I have removed the whole rule, but please double check me as I don't have all of the plugins/extensions installed.

Reply:
I think what you've added resolves the primary issue. Although I do think there is some unnecessary, maybe conflicting and/or redundant CSS affecting the filter items, which could be fixed, that could be tabled for a future sassification. This is a big improvement overall!

Reply:
There are definitely still issues with some of the filter layouts (and additional overly complicated CSS) but that can be followed up separately as I'm sure we'll want to do another sweep after the Chosen changes.